crystal/kemal/shard.yml

```diff
@@ -11,6 +11,6 @@ targets:
 dependencies:
   kemal:
     github: kemalcr/kemal
-    kemal: ~> 0.26.0
+    branch: master
```

Comment:
This should probably be a `commit` until a new release is created.

Reply:
It could .. but I'll not merged it since released :-)
```
kemal:
    github: kemalcr/kemal
    commit: a8c0f09b858162bd13c96663febef5527b322a32
```